### DISCUSSION TITLE
Add atmospheric pressure sensor device class

### DISCUSSION THREAD

**Discussion by emontnemery** [home-assistant/architecture#839] [Entity Models] [answered]
## Proposal
Add an `atmospheric_pressure` sensor device_class to home assistant core and frontend (+ documentation). While there already is a `pressure` device class, the unit for atmospheric pressure is in many cases different. As an example, USCS uses psi for tire pressure and inHg for `atmospheric_pressure`.

Integrations which would benefit from this:
`aemet` , `airly`, `ambient_station`, `bloomsky`, `buienradar`, `meteo_france`, `meteoclimatic`, `nam`, `nws`, `openweathermap`, `rfxtrx`, `tomorrowio`

**Comment by epenet** [comment]
Implementation PR: https://github.com/home-assistant/core/pull/83455

**Comment by emontnemery** [comment, marked as answer]
The proposal is accepted
